pkg/solana/config/multinode.go

```diff
@@ -1,87 +1,163 @@
-	return c.selectionMode
+	return *c.selectionMode
 }
 
 func (c *MultiNode) SyncThreshold() uint32 {
-	return c.syncThreshold
+	return *c.syncThreshold
 }
 
 func (c *MultiNode) NodeIsSyncingEnabled() bool {
-	return c.nodeIsSyncingEnabled
+	return *c.nodeIsSyncingEnabled
 }
 
+func (c *MultiNode) LeaseDuration() time.Duration { return c.leaseDuration.Duration() }
+
 func (c *MultiNode) FinalizedBlockPollInterval() time.Duration {
-	return c.finalizedBlockPollInterval
+	return c.finalizedBlockPollInterval.Duration()
 }
 
-func (c *MultiNode) EnforceRepeatableRead() bool {
-	return c.enforceRepeatableRead
-}
+func (c *MultiNode) EnforceRepeatableRead() bool { return *c.enforceRepeatableRead }
 
-func (c *MultiNode) DeathDeclarationDelay() time.Duration {
-	return c.deathDeclarationDelay
-}
+func (c *MultiNode) DeathDeclarationDelay() time.Duration { return c.deathDeclarationDelay.Duration() }
 
 func (c *MultiNode) NodeNoNewHeadsThreshold() time.Duration {
-	return c.nodeNoNewHeadsThreshold
+	return c.nodeNoNewHeadsThreshold.Duration()
 }
 
 func (c *MultiNode) NoNewFinalizedHeadsThreshold() time.Duration {
-	return c.noNewFinalizedHeadsThreshold
+	return c.noNewFinalizedHeadsThreshold.Duration()
 }
 
-func (c *MultiNode) FinalityDepth() uint32 {
-	return c.finalityDepth
-}
+func (c *MultiNode) FinalityDepth() uint32 { return *c.finalityDepth }
 
-func (c *MultiNode) FinalityTagEnabled() bool {
-	return c.finalityTagEnabled
-}
+func (c *MultiNode) FinalityTagEnabled() bool { return *c.finalityTagEnabled }
 
-func (c *MultiNode) FinalizedBlockOffset() uint32 {
-	return c.finalizedBlockOffset
-}
+func (c *MultiNode) FinalizedBlockOffset() uint32 { return *c.finalizedBlockOffset }
 
 func (c *MultiNode) SetDefaults() {
-	// TODO: Set defaults for MultiNode config https://smartcontract-it.atlassian.net/browse/BCI-4065
-	c.multiNodeEnabled = false
+	c.multiNodeEnabled = ptr(false)
+
+	// Node Configs
+	c.pollFailureThreshold = ptr(uint32(5))
+	c.pollInterval = config.MustNewDuration(10 * time.Second)
+
+	c.selectionMode = ptr(client.NodeSelectionModePriorityLevel)
+
+	c.syncThreshold = ptr(uint32(5))
+
+	// Period at which we verify if active node is still highest block number
+	c.leaseDuration = config.MustNewDuration(time.Minute)
+
+	c.nodeIsSyncingEnabled = ptr(false)
+	c.finalizedBlockPollInterval = config.MustNewDuration(5 * time.Second)
+	c.enforceRepeatableRead = ptr(true)
+	c.deathDeclarationDelay = config.MustNewDuration(10 * time.Second)
+
+	// Chain Configs
+	c.nodeNoNewHeadsThreshold = config.MustNewDuration(10 * time.Second)
+	c.noNewFinalizedHeadsThreshold = config.MustNewDuration(10 * time.Second)
+	c.finalityDepth = ptr(uint32(0))
+	c.finalityTagEnabled = ptr(true)
+	c.finalizedBlockOffset = ptr(uint32(0))
+}
+
+func (c *MultiNode) SetFrom(fs *MultiNode) {
+	if fs.multiNodeEnabled != nil {
+		c.multiNodeEnabled = fs.multiNodeEnabled
+	}
+
+	// Node Configs
+	if fs.pollFailureThreshold != nil {
```

Comment:
Here you could use a helper function or reflection to clean this up. I like the reflection way of doing stuff like this unless you want to manually exclude some of the fields.